### PR TITLE
Math Magicians: tests

### PR DESCRIPTION
**Pull request by fbclh**
**Project requirements**

- Write unit tests for the files operate.js and calculate.js using Jest.
- Create unit tests for all React components:
- Use Jest snapshots to test the components.
- Use React Testing Library to simulate user interaction.
- Run your full tests suite using Jest. All tests should pass.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Math Magicians
 
+Math magicians is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to: Make simple calculations and read a random math-related quote
+
 ## Live Demo
 
 ## Built With
```
Comment (elmar8287) [inline README.md:7]: - [ ] Please, consider fix this sections:
- If you have deployed the project, then add the link for the Live demo
- Add techs with built this project 👍

Comment (elmar8287) [inline README.md:7]: - [ ] Please, consider adding all Authors, as you worked not alone.

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, @fbclh.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the comments under the review._

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (fbclh) [reply README.md:9]: Hi there and thanks, I have made the changes based on your feedback:

- Add the link for the live demo
- Add techs with built this project
- Add all authors

**Review by Kingstalux — APPROVED** [review]
Hi @fbclh and @Alexr16,
# Status: Approved ✔️ ✔️ ✔️ 
 
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966

## Highlights 👏 
✅ All tests are passing 🔥 
✅ Professional README file 💯 
✅ Linters are passing 👍 
✅ Good commit messages 👏 

Good job implementing the required changes 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**